most-time/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@cycle/most-time",
+  "version": "0.10.0",
+  "description": "A time driver designed to enable awesome testing and dev tooling",
+  "license": "MIT",
+  "homepage": "https://cycle.js.org",
+  "bugs": "https://github.com/cyclejs/cyclejs/issues",
+  "repository": "https://github.com/cyclejs/cyclejs/tree/master/time",
+  "author": "Nick Johnstone",
+  "main": "lib/index.js",
+  "module": "lib/es6/index.js",
+  "typings": "lib/index.d.ts",
+  "types": "lib/index.d.ts",
+  "dependencies": {
+    "@cycle/time": "^0.10.0",
+    "most": "*"
+  },
+  "devDependencies": {
+    "@cycle/dom": "17.6",
+    "@types/mocha": "2.2.x",
+    "@types/node": "7.0.x",
+    "garnish": "5.2.x",
+    "markdown-doctest": "0.9.x",
+    "snabbdom-selector": "1.2.x"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "lib/",
+    "dist/",
+    "most.js",
+    "rxjs.js",
+    "most.d.ts",
+    "rxjs.d.ts"
+  ],
+  "browserify-shim": {
+    "most": "global:most"
+  },
+  "scripts": {
+    "start": "../../node_modules/.bin/budo -d example example/index.ts:index.js --live -- -p tsify",
```

Comment:
We could remove `"start"` and this example. Anyway examples should live under the `/examples` directory.